composer/core/evaluator.py

```diff
@@ -141,17 +141,27 @@ def ensure_evaluator(evaluator: Union[Evaluator, DataSpec, Iterable, Dict[str, A
         )
 
 
-def validate_eval_automicrobatching(auto_microbatching: bool, device: Device):
```

Comment:
Doesn't this break the API?

Reply:
Technically yes, but this is mostly an internal validation function -- given we are pre 1.0, I think it's fine to do something like this which cleans up code and almost certainly is not used externally

Reply:
Rather, could we remove this from composer.core if this is the case?

Reply:
Yea good suggestion. Moved to trainer as a private fn